debezium-history-compaction-tool/src/test/java/io/debezium/compaction/tool/service/compaction/KafkaDatabaseHistoryCompactionTest.java

```diff
@@ -60,20 +64,58 @@ public static void stopKafka() {
         }
     }
 
-    @Test
-    public void testConfiguration() throws InterruptedException {
-        if (kafka != null) {
-            System.out.println("is kafka running: " + kafka.isRunning());
+    @Before
+    public void beforeEach() {
+        MySqlPartition source = new MySqlPartition("dbserver", "my-db");
+        this.config = Configuration.empty().edit().with(RelationalDatabaseConnectorConfig.SERVER_NAME, "dbserver").build();
+
+        position = new MySqlOffsetContext(false, true, new TransactionContext(), new MySqlReadOnlyIncrementalSnapshotContext<>(),
+                new SourceInfo(new MySqlConnectorConfig(config)));
+
+        setLogPosition(0);
+        offsets = Offsets.of(source, position);
+
+    }
+
+    @After
+    public void afterEach() {
+        try {
+            if (historyCompaction != null) {
+                historyCompaction.stop();
+            }
         }
-        else {
-            System.out.println("Kafka is not running it's null");
+        finally {
+            historyCompaction = null;
         }
+    }
 
-        // KafkaDatabaseHistoryCompaction databaseHistoryCompaction = new KafkaDatabaseHistoryCompaction("", "", "");
-        //
-        // Configuration dbHistoryConfig = Configuration.create().build();
-        //
-        // databaseHistoryCompaction.configure(dbHistoryConfig, null, DatabaseHistoryListener.NOOP, false);
+    @Test
+    public void shouldStartWithEmptyHistoryTopic() {
+        final String historyTopicName = "my-db-history-topic";
+
+        kafka.createTopic(historyTopicName, 1, 1);
+
+    }
+
+    @Test
+    public void shouldPerformCompactionOnHistoryTopic() {
+        final String bootstrapServers = "localhost:9092";
+        final String historyTopic = "my-db-history-topic";
+        final String compactedHistoryTopic = "my-compacted-db-history-topic";
+
+        historyCompaction = new KafkaDatabaseHistoryCompaction(bootstrapServers, historyTopic, compactedHistoryTopic);
+
+        historyCompaction.configure(config, null, DatabaseHistoryListener.NOOP, false);
+
+        historyCompaction.start();
+
+        historyCompaction.record(offsets.getTheOnlyPartition().getSourcePartition(), offsets.getTheOnlyOffset().getOffset(), new Tables(),
+                new MySqlAntlrDdlParser());
+
+        historyCompaction.stop();
+    }
```

Comment:
In this test, we aren't populating the history topic with any data prior to the compaction.  The goal here was to create the history topic, manually populate it with some records, and run the compaction.  After the compaction had concluded, we wanted to assert that the data in the new topic was what we expected, but that also isn't present in this test case.